move/sh_scripts/move_publish.sh

```diff
@@ -4,7 +4,7 @@ set -e
 
 echo "##### Publishing module #####"
 
-PROFILE=testnet-2
+PROFILE=testnet-5
```

Comment:
If using a profile, it should be provided by the user, since this is a public repository for people to use.

Perhaps you can just use `$PROFILE` and then add instructions to set the variable prior.  I suggest, possibly using either `aptogotchi` or `default` as the default.